src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -2840,20 +2837,19 @@
 									</Storyboard>-->
 								</VisualState>
 								<VisualState x:Name="PointerOver">
```

Comment:
It seems you're fixing only PointerOver, but other visual states appear to be broken as well